src/js/components/modals/CliInstallModal.js

```diff
@@ -18,9 +18,12 @@ class CliInstallModal extends React.Component {
   constructor() {
     super(...arguments);
 
-    this.state = {
-      selectedOS: browserInfo().os
-    };
+    let selectedOS = browserInfo().os;
+    if (Object.values(osTypes).includes(selectedOS) === false) {
```

Comment:
This is fine but syntactically we usually don't do explicit false equality checks - usually just use `!` prefix. Just a #nit.